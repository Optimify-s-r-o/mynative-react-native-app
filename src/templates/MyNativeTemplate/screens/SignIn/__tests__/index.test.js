import * as React from 'react';
import renderer from 'react-test-renderer';
import { SignInTemplate } from '../';

it(`signInTemplate renders correctly`, () => {
  const tree = renderer.create(<SignInTemplate />).toJSON();

  expect(tree).toMatchSnapshot();
});
