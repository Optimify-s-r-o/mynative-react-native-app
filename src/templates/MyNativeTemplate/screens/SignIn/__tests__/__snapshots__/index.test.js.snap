// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`signInTemplate renders correctly 1`] = `
<View>
  <Text>
    test
  </Text>
</View>
`;
